rest-service/manager_rest/es_storage_manager.py

```diff
@@ -152,36 +151,28 @@ def _build_filter_terms_query(filters=None):
         :return: an elasticsearch query string containing the given filters
         """
         terms_lst = []
-        query = None
+        body = {}
+        if pagination:
+            if not skip_size:
+                body['size'] = pagination.get('page_size', DEFAULT_SEARCH_SIZE)
+            if 'offset' in pagination:
+                body['from'] = pagination['offset']
+        elif not skip_size:
```

Comment:
i added this so the "delete deployment" doesn't fail with "unsupported parameter: size"